ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -100,11 +100,12 @@ class LocalAssemblerData
                                  sm.detJ * wp.getWeight();
 
             // Darcy velocity only computed for output.
-            auto const darcy_velocity = -(k * sm.dNdx *
-                Eigen::Map<const NodalVectorType>(local_x.data(), ShapeFunction::NPOINTS)
-                ).eval();
+            NodalVectorType const darcy_velocity =
```

Comment:
Thanks! I did miss this one. The compiler missed too, because we don't compile PRs with fixed size matrices...